components/views.py

```diff
@@ -56,11 +62,11 @@ def list(self, request, *args, **kwargs):
 
 
 class ComponentTypeListView(generics.ListAPIView):
-    queryset = Component.objects.order_by().values_list("type").distinct()
+    queryset = Component.objects.values_list("type")
```

Comment:
Not entirely sure what was happening here to cause duplicates after applying the `union`, but removing the `order_by` and `distinct` calls fixes the issue. They weren't really useful in this case anyways.